app/views/shared/_navbar.html.erb

```diff
@@ -1,8 +1,31 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
-  <%= link_to root_path, class: "navbar-brand" do %>
-    <i class="fas fa-address-book"></i>
-    <% end %>
+  <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
```

Comment:
These modifications also seem to be in your previous pull request. Was there any change? If not, please check the comments on the other pull request